app/src/main/java/com/kickstarter/ui/viewholders/ProjectViewHolder.java

```diff
@@ -143,82 +132,333 @@
     super(view);
+    this.viewModel = new ProjectHolderViewModel.ViewModel(environment());
     this.delegate = delegate;
-    this.context = view.getContext();
-    this.ksCurrency = environment().ksCurrency();
     this.ksString = environment().ksString();
 
     ButterKnife.bind(this, view);
+
+    this.viewModel.outputs.avatarPhotoUrl()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(url ->
+        Picasso.with(context())
+          .load(url)
+          .transform(new CircleTransformation())
+          .into(this.avatarImageView)
+      );
+
+    this.viewModel.outputs.backersCountTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this.backersCountTextView::setText);
+
+    this.viewModel.outputs.backingViewGroupIsGone()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(ViewUtils.setGone(this.backingViewGroup));
+
+    this.viewModel.outputs.blurbTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(blurb -> this.blurbTextView.setText(Html.fromHtml(TextUtils.htmlEncode(blurb))));
+
+    this.viewModel.outputs.categoryTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this.categoryTextView::setText);
+
+    this.viewModel.outputs.commentsCountTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this.commentsCountTextView::setText);
+
+    this.viewModel.outputs.creatorNameTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(name ->
+        this.creatorNameTextView.setText(
+          Html.fromHtml(this.ksString.format(this.byCreatorString, "creator_name", TextUtils.htmlEncode(name))))
+      );
+
+    this.viewModel.outputs.deadlineCountdownTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this.deadlineCountdownTextView::setText);
+
+    this.viewModel.outputs.featuredTextViewRootCategory()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(c ->
+        this.featuredTextView.setText(this.ksString.format(this.featuredInString, "category_name", c))
+      );
+
+    this.viewModel.outputs.featuredViewGroupIsGone()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(ViewUtils.setGone(this.featuredViewGroup));
+
+    this.viewModel.outputs.goalStringForTextView()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::setGoalTextView);
+
+    this.viewModel.outputs.locationTextViewText()
+      .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this.locationTextView::setText);
+
+    this.viewModel.outputs.projectOutput()
+      .subscribe(p -> {
+        // todo: break down these helpers
```

Comment:
this is what i meant re: not being sure how to handle landscape and a11y specific outputs where it's kinda inevitable to just pass along the same project output to...

Reply:
yeah agreed that's a little cleaner than invoking a chain of helpers